test/app/lib/cinematic/commandRunner.spec.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-env jasmine */
+import CommandRunner from '../../../../app/views/play/cinematic/Command/CommandRunner'
+import AbstractCommand from '../../../../app/views/play/cinematic/Command/AbstractCommand'
+import Promise from 'bluebird'
+const utils = require('../../utils.coffee')
+
+Promise.config({
+  cancellation: true
+})
+
+const sleep = ms => new Promise((resolve, reject) => setTimeout(resolve, ms))
```

Comment:
Question for my own understanding: Because this test runs in node, it doesn't suffer from the issues described in the `sleep` command above?